src/consent-manager/banner.js

```diff
@@ -36,19 +36,12 @@ const P = styled('p')`
   }
 `
 
-const CloseButton = styled('button')`
+const AcceptAllButton = styled('button')`
```

Comment:
We can keep name `CloseButton` with intention to merge our changes to segment repo. If we don't change name it could be easier, what do you think?

Reply:
makes total sense! Wasn't aware of how much was our banner vs their banner.